.github/workflows/checks.yml

```diff
@@ -99,7 +99,7 @@ jobs:
       - name: Install dependencies
         run: npm clean-install
       - name: Check licenses
-        run: npm run license-check
+        run: npm run audit:licenses
```

Comment:
I renamed `license-check` to `audit:licenses` in part because I think it sounds a bit better, but primarily because it is listed under "Auditing" in the Contributing Guidelines. However, in contrast to deprecation warnings and vulnerabilities, the license check should always pass so maybe it's better to group it under "Vetting"?

Reply:
See f02f88b11c1489378450c9b19a318c76c190cd53